### ISSUE TITLE
activate a environment path don't contain the new environment's Lib dir, resulting error!!!

### ISSUE THREAD

**Issue by liyi193328** [conda/conda#1829]
When I install a python2.7 environment in my win10+python3.4 environment using "conda create -n python2 python=2.7";
After installtion and " activate python2", I input "conda list" , resulting errors, as the following, I don't know why,please help me!
### [python2] C:\Users\liyi1>conda info

Current conda install:

```
         platform : win-64
    conda version : 3.18.5
```

  conda-build version : 1.11.0
       python version : 3.4.3.final.0
     requests version : 2.8.1
     root environment : D:\Anaconda3  (writable)
  default environment : D:\Anaconda3\envs\python2
     envs directories : D:\Anaconda3\envs
        package cache : D:\Anaconda3\pkgs
         channel URLs : https://repo.continuum.io/pkgs/free/win-64/
                        https://repo.continuum.io/pkgs/free/noarch/
                        https://repo.continuum.io/pkgs/pro/win-64/
                        https://repo.continuum.io/pkgs/pro/noarch/
          config file : None
    is foreign system : False
### [python2] C:\Users\liyi1>conda list

 packages in environment at D:\Anaconda3\envs\python2:

  File "D:\Anaconda3\Lib\site.py", line 178
    file=sys.stderr)
        ^
SyntaxError: invalid syntax
An unexpected error has occurred, please consider sending the
following traceback to the conda GitHub issue tracker at:

```
https://github.com/conda/conda/issues
```

Include the output of the command 'conda info' in your report.

Traceback (most recent call last):
  File "D:\Anaconda3\Scripts\conda-script.py", line 4, in <module>
    sys.exit(main())
  File "D:\Anaconda3\lib\site-packages\conda\cli\main.py", line 195, in main
    args_func(args, p)
  File "D:\Anaconda3\lib\site-packages\conda\cli\main.py", line 202, in args_fun
c
    args.func(args, p)
  File "D:\Anaconda3\lib\site-packages\conda\cli\main.py", line 202, in args_fun
c
    args.func(args, p)
  File "D:\Anaconda3\lib\site-packages\conda\cli\main_list.py", line 246, in exe
cute
    json=args.json, show_channel_urls=args.show_channel_urls)
  File "D:\Anaconda3\lib\site-packages\conda\cli\main_list.py", line 177, in pri
nt_packages
    add_pip_installed(prefix, installed, json=json)
  File "D:\Anaconda3\lib\site-packages\conda\pip.py", line 108, in add_pip_insta
lled
    for pip_pkg in installed(prefix, output=output):
  File "D:\Anaconda3\lib\site-packages\conda\pip.py", line 49, in installed
    args = pip_args(prefix)
  File "D:\Anaconda3\lib\site-packages\conda\pip.py", line 28, in pip_args
    pip_version = subprocess.check_output(ret + ['-V']).decode('utf-8').split()[
1]
  File "D:\Anaconda3\Lib\subprocess.py", line 620, in check_output
    raise CalledProcessError(retcode, process.args, output=output)
subprocess.CalledProcessError: Command '['D:\Anaconda3\envs\python2\python.e
xe', 'D:\Anaconda3\envs\python2\Scripts\pip-script.py', '-V']' returned non
-zero exit status 1

**Comment by lesiak** [comment]
I had similar problem on Win7.
In my case, the problem was PYTHONHOME environment variable.
After removing it, the problem disappered
https://docs.python.org/2/using/cmdline.html#envvar-PYTHONHOME

**Comment by liyi193328** [comment]
@lesiak thanks

**Comment by kalefranz** [comment]
Fixed as of conda 4.1.
